api/HTMLElement.json

```
@@ -2354,6 +2423,120 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "details_elements": {
+          "__compat": {
+            "description": "<code>toggle</code> event fires at details elements",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/toggle_event",
+            "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-toggle",
+            "tags": [
+              "web-features:details"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "36"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "49"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "10.1"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "dialog_elements": {
+          "__compat": {
+            "description": "<code>toggle</code> event fires at dialog elements",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/toggle_event",
+            "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-toggle",
+            "support": {
+              "chrome": {
+                "version_added": false
```

https://chromestatus.com/roadmap This should probably be `132` as it is slated to ship, barring any major blockers such as compat risk, which would be quite low for a non-bubbling event IMO.

```suggestion
                "version_added": "132"
```